File: federaproApi/src/main/java/com/example/federaproapi/karting/dto/CrearKartingCompeticionDTO.java
package com.example.federaproapi.karting.dto;

import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;
import java.util.List;

@Getter
@Setter
public class CrearKartingCompeticionDTO {
    public String nombre;
    public String tipo;
    public String categoria;
    public LocalDate fechaInicio;
    public LocalDate fechaFin;
    public List<Integer> equiposIds;
}

File: FederaProDesktop/Karting/DTOs/CrearKartingCompeticionDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace FederaProDesktop.Karting.DTOs
{
    public class CrearKartingCompeticionDto
    {
        public string nombre { get; set; }
        public string tipo { get; set; }
        public string categoria { get; set; }
        public DateTime fechaInicio { get; set; }
        public DateTime fechaFin { get; set; }
        public List<long> equiposIds { get; set; }
    }
}
